frontend/src/pages/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -0,0 +1,49 @@
+import { type ChangeEvent } from 'react';
+
+import sendIcon from '#/assets/img/send-icon.svg';
+import { useCallback, useState } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type ChatFooterProperties = {
+  onSend: (text: string) => void;
+};
+
+const ChatFooter: React.FC<ChatFooterProperties> = ({ onSend }) => {
+  const [text, setText] = useState('');
+
+  const textHandler = useCallback(
```

Comment:
According to the rule A10 component functions should be named with handle prefix, like handleInputChange